babel.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = (api) => {
           exclude:
             process.env.NODE_ENV === "test" ? [] : ["proposal-dynamic-import"],
           targets: {
-            node: "12.20.0",
+            node: "14.16.0",
```

Comment:
@snitin315 Let's fix it to `14.15.0`

Reply:
@alexander-akait done, updated deps too.